entry_test.go

```diff
@@ -34,6 +38,9 @@ func TestEntryWithError(t *testing.T) {
 
 func TestEntryPanicln(t *testing.T) {
 	errBoom := fmt.Errorf("boom time")
+	logger := New()
+	logger.Formatter = &TextFormatter{DisableTimestamp: true, DisableColors: true}
```

Comment:
I would advise to use a json formatter, it is way easier to check a json output.

Reply:
While I did, as you suggested, switch to the json formatter in my new test(s), this was an existing test from 4 years ago. (My minor change, that I'm happy to revert, simply alternated the order of operations setting up the test. I believe I did this for readability.)